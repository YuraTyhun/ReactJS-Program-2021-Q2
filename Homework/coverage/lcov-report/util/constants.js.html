
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for util/constants.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../index.html">All files</a> / <a href="index.html">util</a> constants.js</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>6/6</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>6/6</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line high'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a>
<a name='L36'></a><a href='#L36'>36</a>
<a name='L37'></a><a href='#L37'>37</a>
<a name='L38'></a><a href='#L38'>38</a>
<a name='L39'></a><a href='#L39'>39</a>
<a name='L40'></a><a href='#L40'>40</a>
<a name='L41'></a><a href='#L41'>41</a>
<a name='L42'></a><a href='#L42'>42</a>
<a name='L43'></a><a href='#L43'>43</a>
<a name='L44'></a><a href='#L44'>44</a>
<a name='L45'></a><a href='#L45'>45</a>
<a name='L46'></a><a href='#L46'>46</a>
<a name='L47'></a><a href='#L47'>47</a>
<a name='L48'></a><a href='#L48'>48</a>
<a name='L49'></a><a href='#L49'>49</a>
<a name='L50'></a><a href='#L50'>50</a>
<a name='L51'></a><a href='#L51'>51</a>
<a name='L52'></a><a href='#L52'>52</a>
<a name='L53'></a><a href='#L53'>53</a>
<a name='L54'></a><a href='#L54'>54</a>
<a name='L55'></a><a href='#L55'>55</a>
<a name='L56'></a><a href='#L56'>56</a></td><td class="line-coverage quiet"><span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">5x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">export const genres = [
    'Action', 
    'Comedy', 
    'Crime', 
    'Drama', 
    'Music', 
    'Romance',
    'Adventure',
    'Family',
    'Fantasy',
    'Science Fiction',
    'Thriller',
    'Documentary',
    'Horror',
    'History',
    'Western'
];
&nbsp;
export const sortBy = [
    { id: 1, value: 'release_date', label: 'RELEASE DATE' },
    { id: 2, value: 'runtime', label: 'RUNTIME' },
    { id: 3, value: 'vote_average', label: 'RATING' },
];
&nbsp;
export const formInputsList = [
    { id: 1, label: "TITLE", name: "title", placeholder: "Movie title here", type: "text" },
    { id: 2, label: "RELEASE DATE", name: "release_date", placeholder: "Select Date", type: "date" },
    { id: 3, label: "MOVIE URL", name: "poster_path", placeholder: "Movie URL here", type: "text" },
    { id: 4, label: "GENRE", fieldName: "genres"},
    { id: 5, label: "OVERVIEW", name: "overview", placeholder: "Overview here", type: "text" },
    { id: 6, label: "RUNTIME", name: "runtime", placeholder: "Runtime here", type: "text" },
    { id: 7, label: "BUDGET", name: "budget", placeholder: "Budget here", type: "text" },
    { id: 8, label: "REVENUE", name: "revenue", placeholder: "Revenue here", type: "text" },
    { id: 9, label: "TAGLINE", name: "tagline", placeholder: "Tagline here", type: "text" },
    { id: 10, label: "VOTE AVERAGE", name: "vote_average", placeholder: "Vote average here", type: "text" },
    { id: 11, label: "VOTE COUNT", name: "vote_count", placeholder: "Vote count here", type: "text" }
];
&nbsp;
export const BASE_URL = 'http://localhost:4000/movies';
&nbsp;
export const URL_REGEX = /((https?):\/\/)?(www.)?[a-z0-9-]+(\.[a-z]{2,}){1,3}(#?\/?[a-zA-Z0-9#-]+)*\/?(\?[a-zA-Z0-9-_]+=[a-zA-Z0-9-%]+&amp;?)?$/;
&nbsp;
export const mockMovie = {
    id: 1,
    title: 'Avengers: War of Infinity',
    budget: 250000000,
    genres: ['Action', 'Adventure'],
    overview: 'Avengers: Infinity War is a 2018 American superhero film.',
    poster_path: '',
    release_date: '2018-04-28',
    revenue: 1850000000,
    runtime: 155,
    tagline: 'Tagline',
    vote_average: 8.5,
    vote_count: 175896
};</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Tue Aug 10 2021 14:27:46 GMT+0200 (GMT+02:00)
            </div>
        </div>
        <script src="../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../sorter.js"></script>
        <script src="../block-navigation.js"></script>
    </body>
</html>
    