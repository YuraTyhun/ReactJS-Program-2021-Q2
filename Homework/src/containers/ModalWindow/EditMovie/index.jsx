import React from 'react';
import { Formik, Form } from 'formik';
import PropTypes from 'prop-types';
import { useSelector } from 'react-redux';

import MovieForm from '../../MovieForm';

const EditMovie = ({ modalTitle, movie: { id, title, releaseDate, posterUrl, genres, overview, runtime } }) => {

    const activeModal = useSelector((state) => state.movie.activeModal);

    const initialValues = {
        id,
        title,
        releaseDate,
        posterUrl,
        genres,
        overview,
        runtime
    };

    const submitForm = (values) => {
        console.log(values);
    }

    return activeModal === 'edit' && (
        <Formik initialValues={initialValues} onSubmit={submitForm}>
            {(formik) => (
                <Form>
                    <h1 className="modal-window-title">{modalTitle}</h1>
                    <MovieForm formik={formik} />
                </Form>
            )}
        </Formik>
    )
}

EditMovie.propTypes = {
    modalTitle: PropTypes.string.isRequired,
    movie: PropTypes.shape({
        id: PropTypes.number,
        title: PropTypes.string,
        releaseDate: PropTypes.string,
        posterUrl: PropTypes.string,
        genres: PropTypes.arrayOf(PropTypes.string),
        overview: PropTypes.string,
        runtime: PropTypes.number
    }).isRequired
};

export default EditMovie;