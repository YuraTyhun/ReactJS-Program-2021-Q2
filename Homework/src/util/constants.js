export const genres = [
    'Action', 
    'Comedy', 
    'Crime', 
    'Drama', 
    'Music', 
    'Romance',
    'Adventure',
    'Family',
    'Fantasy',
    'Science Fiction',
    'Thriller',
    'Documentary',
    'Horror',
    'History',
    'Western'
];

export const sortBy = [
    { id: 1, value: 'release_date', label: 'RELEASE DATE' },
    { id: 2, value: 'runtime', label: 'RUNTIME' },
    { id: 3, value: 'vote_average', label: 'RATING' },
];

export const formInputsList = [
    { id: 1, label: "TITLE", name: "title", placeholder: "Movie title here" },
    { id: 2, label: "RELEASE DATE", name: "release_date", placeholder: "Select Date", type: "date" },
    { id: 3, label: "MOVIE URL", name: "poster_path", placeholder: "Movie URL here" },
    { id: 4, label: "GENRE", fieldName: "genres" },
    { id: 5, label: "OVERVIEW", name: "overview", placeholder: "Overview here" },
    { id: 6, label: "RUNTIME", name: "runtime", placeholder: "Runtime here" },
    { id: 7, label: "BUDGET", name: "budget", placeholder: "Budget here" },
    { id: 8, label: "REVENUE", name: "revenue", placeholder: "Revenue here" },
    { id: 9, label: "TAGLINE", name: "tagline", placeholder: "Tagline here" },
    { id: 10, label: "VOTE AVERAGE", name: "vote_average", placeholder: "Vote average here" },
    { id: 11, label: "VOTE COUNT", name: "vote_count", placeholder: "Vote count here" }
];

export const BASE_URL = 'http://localhost:4000/movies';