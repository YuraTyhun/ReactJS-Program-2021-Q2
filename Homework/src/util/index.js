export const sortList = (arr = [], sortBy) => {
    return sortBy === 'releaseDate' ?
        arr.sort((a,b) => new Date(b[sortBy]).getFullYear() - new Date(a[sortBy]).getFullYear()) :
        arr.sort((a,b) => b[sortBy] - a[sortBy]);  
}