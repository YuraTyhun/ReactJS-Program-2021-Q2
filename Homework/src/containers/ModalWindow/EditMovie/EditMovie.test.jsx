import React from 'react';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import EditMovie from './index';
import { mockMovie } from '../../../util/constants';

it('renders addMovie form correctly', () => {
  const mockStore = configureMockStore();
  const initState = {
    movie: {
      activeModal: 'edit'
    }
  };

  const store = mockStore(initState);
  const addMovie = renderer.create(
    <Provider store={store}>
      <EditMovie modalTitle="EDIT MOVIE" movie={mockMovie} />
    </Provider>
  );

  expect(addMovie).toMatchSnapshot();
});
