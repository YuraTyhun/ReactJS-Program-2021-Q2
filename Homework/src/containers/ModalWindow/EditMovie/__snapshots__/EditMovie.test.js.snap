// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders addMovie form correctly 1`] = `
<form
  action="#"
  onReset={[Function]}
  onSubmit={[Function]}
>
  <h1
    className="modal-window-title"
  >
    EDIT MOVIE
  </h1>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="id"
    >
      MOVIE ID
    </label>
    <input
      className="form-input-field id-field"
      name="id"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder=""
      readOnly={true}
      value={1}
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="title"
    >
      TITLE
    </label>
    <input
      className="form-input-field"
      name="title"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Movie title here"
      type="text"
      value="Avengers: War of Infinity"
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="release_date"
    >
      RELEASE DATE
    </label>
    <input
      className="form-input-field"
      name="release_date"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Select Date"
      type="date"
      value="2018-04-28"
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="poster_path"
    >
      MOVIE URL
    </label>
    <input
      className="form-input-field"
      name="poster_path"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Movie URL here"
      type="text"
      value=""
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="genres"
    >
      GENRE
    </label>
    <div
      aria-expanded={false}
      aria-owns={null}
      className="MuiAutocomplete-root form-input-field MuiAutocomplete-hasClearIcon MuiAutocomplete-hasPopupIcon"
      name="genres"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      role="combobox"
    >
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
      >
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiAutocomplete-inputRoot MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
          onClick={[Function]}
        >
          <div
            className="MuiChip-root MuiAutocomplete-tag MuiChip-deletable"
            data-tag-index={0}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            role="button"
            tabIndex={-1}
          >
            <span
              className="MuiChip-label"
            >
              Action
            </span>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiChip-deleteIcon"
              focusable="false"
              onClick={[Function]}
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"
              />
            </svg>
          </div>
          <div
            className="MuiChip-root MuiAutocomplete-tag MuiChip-deletable"
            data-tag-index={1}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            role="button"
            tabIndex={-1}
          >
            <span
              className="MuiChip-label"
            >
              Adventure
            </span>
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiChip-deleteIcon"
              focusable="false"
              onClick={[Function]}
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2zm5 13.59L15.59 17 12 13.41 8.41 17 7 15.59 10.59 12 7 8.41 8.41 7 12 10.59 15.59 7 17 8.41 13.41 12 17 15.59z"
              />
            </svg>
          </div>
          <input
            aria-activedescendant={null}
            aria-autocomplete="list"
            aria-controls={null}
            aria-invalid={false}
            autoCapitalize="none"
            autoComplete="off"
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
            disabled={false}
            id="genres"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onMouseDown={[Function]}
            placeholder="Select genre"
            required={false}
            spellCheck="false"
            type="text"
            value=""
          />
          <div
            className="MuiAutocomplete-endAdornment"
          >
            <button
              aria-label="Clear"
              className="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-clearIndicator MuiAutocomplete-clearIndicatorDirty"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={-1}
              title="Clear"
              type="button"
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                  />
                </svg>
              </span>
            </button>
            <button
              aria-label="Open"
              className="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-popupIndicator"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={-1}
              title="Open"
              type="button"
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </span>
            </button>
          </div>
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
            style={
              Object {
                "paddingLeft": 8,
              }
            }
          >
            <legend
              className="PrivateNotchedOutline-legend-2"
              style={
                Object {
                  "width": 0.01,
                }
              }
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <span
      className="form-input-error"
    />
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="overview"
    >
      OVERVIEW
    </label>
    <input
      className="form-input-field"
      name="overview"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Overview here"
      type="text"
      value="Avengers: Infinity War is a 2018 American superhero film."
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="runtime"
    >
      RUNTIME
    </label>
    <input
      className="form-input-field"
      name="runtime"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Runtime here"
      type="text"
      value={155}
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="budget"
    >
      BUDGET
    </label>
    <input
      className="form-input-field"
      name="budget"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Budget here"
      type="text"
      value={250000000}
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="revenue"
    >
      REVENUE
    </label>
    <input
      className="form-input-field"
      name="revenue"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Revenue here"
      type="text"
      value={1850000000}
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="tagline"
    >
      TAGLINE
    </label>
    <input
      className="form-input-field"
      name="tagline"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Tagline here"
      type="text"
      value="Tagline"
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="vote_average"
    >
      VOTE AVERAGE
    </label>
    <input
      className="form-input-field"
      name="vote_average"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Vote average here"
      type="text"
      value={8.5}
    />
    
  </div>
  <div
    className="form-input-container"
  >
    <label
      className="form-input-label"
      htmlFor="vote_count"
    >
      VOTE COUNT
    </label>
    <input
      className="form-input-field"
      name="vote_count"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder="Vote count here"
      type="text"
      value={175896}
    />
    
  </div>
  <div
    className="modal-window-action"
  >
    <button
      className="modal-window-action-btn"
      disabled={false}
      onClick={[Function]}
      type="reset"
    >
      RESET
      
    </button>
    <button
      className="modal-window-action-btn"
      disabled={false}
      onClick={[Function]}
      type="submit"
    >
      SAVE
      
    </button>
  </div>
</form>
`;
